nodeServerless/Local CE SAM/CE_lambda_functions/card_get.js

```diff
@@ -0,0 +1,62 @@
+/**
+ *
+ * Event doc: https://docs.aws.amazon.com/apigateway/latest/developerguide/set-up-lambda-proxy-integrations.html#api-gateway-simple-proxy-for-lambda-input-format
+ * @param {Object} event - API Gateway Lambda Proxy Input Format
+ *
+ * Context doc: https://docs.aws.amazon.com/lambda/latest/dg/nodejs-prog-model-context.html 
+ * @param {Object} context
+ *
+ * Return doc: https://docs.aws.amazon.com/apigateway/latest/developerguide/set-up-lambda-proxy-integrations.html
+ * @returns {Object} object - API Gateway Lambda Proxy Output Format
+ * 
+ */
+
+let response;
+var AWS = require('aws-sdk');
+AWS.config.update({
+    region: "us-west-2"
+})
+
+let docClient = new AWS.DynamoDB.DocumentClient();
+// let ddb = new AWS.DynamoDB();
+
+exports.lambdaHandler = async (event, context) => {
```

Comment:
It's up to you, but I prefer promises in this case. try/catch is kind of a code ick. 

It might be cleaner to write this of the form:

```
exports.lambdaHandler = async (event, context) => {
if (!validate(event)) {
  return {
    statusCode: 400,
    error: `Expected input of the form...`
  };
}

return docClient.get({
  // put the whole payload inline
}).promise().then((data) => {
  return { statusCode: 200, body: JSON.stringify(data) };
}).catch(error) => {
  return {
    statusCode: 500,
    error: `Could not fetch: ${error.stack}`
  };
});

```

Reply:
By "put the whole payload inline" I assume you mean define my params var there?
```
return docClient.get({
     var params = {
        TableName: "BulkMagicCards",
        Key: {
            'scryfallid' : id,
            'name' : name
        }      
    };
}).promise().then...
```

Reply:
Yeah, almost.

```
return docClient.get({
    TableName: "BulkMagicCards",
    Key: {
        'scryfallid' : id,
        'name' : name
    }      
}).promise().then...

```